### PR TITLE
Get Settlement info

### PR DESCRIPTION
**Pull request by marctemp**
https://eaflood.atlassian.net/browse/SFI-2319

### CODE REVIEW

File: package.json
Comment (AbidemiAdio) [inline package.json:3]: This will conflict with pull/11 after merge. Also why 0.7.1 but not 0.8.0 ?

Comment (marctemp) [reply package.json:3]: Based version off `main` as we dnt _know_ (altho like 99% sure!) which PR will merge 1st, could rebase if needs be

Did patch as functionality isn't called anywhere yet

Happy to bump as minor and upto `0.8.0` to avoid merge conflicts

**Review by johnwatson484 — APPROVED** [review]

**Pull request merged**
